gateway-util-common/src/main/java/org/apache/knox/gateway/audit/log4j/audit/Log4jAuditor.java

```diff
@@ -52,8 +51,8 @@
   }
 
   public Log4jAuditor( String loggerName, String componentName, String serviceName ) {
-    logger = Logger.getLogger( loggerName );
-    logger.setAdditivity( false );
+    logger = (Logger) LogManager.getLogger( loggerName );
```

Comment:
Is this cast necessary?

Reply:
There is no setAdditivity method in `org.apache.logging.log4j.Logger`, only in `org.apache.logging.log4j.core.Logger`.

This is how the the Log4j's compatibility API does this as well:

https://github.com/apache/logging-log4j2/blob/f630e16ecf196b5a7097a224a7997801b819ddd0/log4j-1.2-api/src/main/java/org/apache/log4j/legacy/core/CategoryUtil.java#L60